cmd/root.go

```diff
@@ -193,33 +197,58 @@ func getWriter(outputFile string) (io.Writer, error) {
 func init() {
 	RootCmd.PersistentFlags().StringVarP(&LogLevel, "verbosity", "v", "warning", "This flag controls the verbosity of container-diff.")
 	RootCmd.PersistentFlags().StringVarP(&format, "format", "", "", "Format to output diff in.")
+	RootCmd.PersistentFlags().VarP(&skipTsVerifyRegistries, "skip-tls-verify-registry", "", "Insecure registry ignoring TLS verify to push and pull. Set it repeatedly for multiple registries.")
+	RootCmd.PersistentFlags().VarP(&registriesCertificates, "registry-certificate", "", "Use the provided certificate for TLS communication with the given registry. Expected format is 'my.registry=/path/to/the/server/certificate'.")
 	pflag.CommandLine.AddGoFlagSet(goflag.CommandLine)
 }
 
-// Define a type named "diffSlice" as a slice of strings
-type diffTypes []string
+// Define a type named "multiValueFlag" as a slice of strings
+type multiValueFlag []string
 
 // Now, for our new type, implement the two methods of
 // the flag.Value interface...
 // The first method is String() string
-func (d *diffTypes) String() string {
-	return strings.Join(*d, ",")
+func (f *multiValueFlag) String() string {
+	return strings.Join(*f, ",")
 }
 
 // The second method is Set(value string) error
-func (d *diffTypes) Set(value string) error {
+func (f *multiValueFlag) Set(value string) error {
 	// Dedupe repeated elements.
-	for _, t := range *d {
+	for _, t := range *f {
 		if t == value {
 			return nil
 		}
 	}
-	*d = append(*d, value)
+	*f = append(*f, value)
+	return nil
+}
+
+func (f *multiValueFlag) Type() string {
+	return "multiValueFlag"
+}
+
+type keyValueFlag map[string]string
+
+func (f *keyValueFlag) String() string {
+	result := []string{}
+	for key := range *f {
+		result = append(result, fmt.Sprintf("%s=%s", key, (*f)[key]))
+	}
```

Comment:
since you're using the values directly here, you can include them in the `range` call and avoid directly accessing:

```golang
for key, value := range *f {
  result = append(result, fmt.Sprintf("%s=%s", key, value))
}
```